docs/getting-started.md

```diff
@@ -0,0 +1,184 @@
+# Getting Started
+
+* TOC
+{:toc}
+
+These guide(s) will show you how to get started with your favourite container orchestrator as well as Kedge!
+
+There are three different guides depending on your container orchestrator as well as operating system.
+
+For beginners and the most compatibility, follow the __Minikube and Kedge__ guide.
+
+## Minikube and Kedge
+
+In this guide, we'll deploy a sample kedge file `httpd.yaml` to a Kubernetes cluster.
+
+Requirements:
+  - [minikube](https://github.com/kubernetes/minikube)
```

Comment:
put the downlaod instructions link!

Reply:
If people follow the link, they'll find the installation instructions :)